src/conf.h

```diff
@@ -244,4 +251,11 @@
 
 #define SEARCH_VELOCITY 0.3
 
+//************UPDATE THESE VALUES ****************//
+#define APAC_TARGET_X 4
+#define APAC_TARGET_Y 4
```

Comment:
The competition maze is 16x16, and the finish is the middle 4 squares. So (7, 7) or (8, 8) would work. I think in the past we did (7, 7).